src/Client.php

```diff
@@ -79,6 +80,22 @@ public function getProject(): Project
         );
     }
 
+    public function updateProjectServerVersion(string $projectId, string $serverVersion): Project
```

Comment:
Here, we could maybe pass the whole Project object to this function, but this seem more handy to me.